labelbox/data/annotation_types/data/tiled_image.py

```diff
@@ -28,6 +28,11 @@
 logger = logging.getLogger(__name__)
 
 
+class AllVectorTools():
+    """Used for the purpose of type hinting"""
+    value: Union[Point, Line, Rectangle, Polygon] = None
```

Comment:
You can just do:
`VectorTool = Union[Point, Line,Rectangle, Polygon]`

Reply:
Ah got it, thanks. Will update